qa/integration-tests/src/test/java/io/camunda/zeebe/it/smoke/StandaloneBrokerIT.java

```diff
@@ -50,6 +50,7 @@ void beforeEach() {
 
   /** A simple smoke test to ensure the broker starts and can perform basic functionality. */
   @Test
+  @SmokeTest
```

Comment:
The `@SmokeTest` annotation is specifically related to JUnit tests (which are all annotated by `@Test` by definition). And the name SmokeTest already states it is a Test. 

🔧 Please push `@Test` up into `@SmokeTest`

Reply:
I was reluctant to do this in case some smoke tests would be Junit4 tests - however since we do want to migrate to junit 5 completely, I guess this isn't that relevant :smile: